site/src/components/select.md

```diff
@@ -0,0 +1,111 @@
+---
+title: Select
+description: The select component in the Atlas Design System
+template: standard
+figmaEmbed: https://www.figma.com/embed?embed_host=share&url=https%3A%2F%2Fwww.figma.com%2Ffile%2FuVA2amRR71yJZ0GS6RI6zL%2F%25F0%259F%258C%259E-Atlas-Design-Library%3Fnode-id%3D1488%253A35182
+---
+
+# Select
+
+The Select element represents a control that provides a menu of options from which the user can make a single selection.
+
+## Usage
+
+Here is an example of a standard select usage paired with a label.
+
+```html
+<label class="label margin-bottom-xxs" for="select-id-1">Label</label>
+<div class="select">
+	<select name="select-1" id="select-id-1">
+		<option value="">Choose an option</option>
+		<option value="option-1">Option 1</option>
+		<option value="option-2">Option 2</option>
+		<option value="option-3">Option 3</option>
+	</select>
+</div>
+```
+
+### Multiple select
+
+There is an option to style a multiple select dropdowns by using the `.select-multiple` class in conjunction with `.select` and adding the `multiple` attribute to the `<select>` tag.
+
+Hold down `control` button on Windows/`command` button on Mac to select multiple options.
```

Comment:
suggestion: I wonder if this is better to use the actual label on the keyboard "`ctrl` key on Windows / `cmd` key on Mac".

Reply:
Updated